lib/improver/tests/utilities/test_TemporalInterpolation.py

```diff
@@ -409,6 +409,14 @@ def setUp(self):
         cubes_ens = iris.cube.CubeList([cube_time_0_ens, cube_time_1_ens])
         self.cube_ens = cubes_ens.merge_cube()
 
+        # set location
```

Comment:
I agree with Gavin. It would probably be better to update the times and expected values rather than keep the old values and swap the lats/lons of the cubes.